builds/linux/rhel/9/variables.pkr.hcl

```diff
@@ -10,8 +10,24 @@
 //  BLOCK: variable
 //  Defines the input variables.
 
+// Additional yum repositories
+variable "yum_repositories" {
+  type        = list(object({
+    name = string
+    url = string
+  }))
+  description = "Additional yum repositories"
```

Comment:
```suggestion
  description = "Additional repositories."
```

Reply:
yum_repositories should not have been in this pull, it removed it